coral-spark/src/test/java/com/linkedin/coral/spark/TestUtils.java

```diff
@@ -193,6 +193,9 @@ public static void initializeViews() throws HiveException, MetaException {
 
     run(driver,
         "CREATE TABLE IF NOT EXISTS union_table(foo uniontype<int, double, array<string>, struct<a:int,b:string>>)");
+
+    // Recursive union case
+    run(driver, "CREATE TABLE IF NOT EXISTS recursive_union(a uniontype<int, uniontype<int, double>>)");
```

Comment:
Could you please add test case for `uniontype<struct<uniontype>>`? We found this case is error prone and better to have unit test to guard it.

Reply:
+1, and I think `nested_union` is better than `recursion_union`?